src/entity/FavouritesList.java

```diff
@@ -24,11 +23,18 @@ public void remove(String restaurantID){
 
     @Override
     public String toString(){
+        if (favouritesList.isEmpty()){
+            return "";
+        }
         StringBuilder result = new StringBuilder();
         for (Restaurant favourite : favouritesList){
             result.append(favourite.getRestaurantID()).append(",");
         }
         result.delete(result.length() - 1, result.length());
         return result.toString();
     }
```

Comment:
Using join method can help achieve your goal, making your intentions clearer to the readers in this context.
```java
List<String> restaurantIds = new ArrayList<>();
for (Restaurant favourite : favouritesList) {
    restaurantIds.add(favourite.getRestaurantID());
}
String result = String.join(",", restaurantIds);
```

Reply:
Good Idea. I don't actually use this method right now, but I kept it in if we ever want to improve the UI of the View Favourites View.